src/main/cpp/mechanisms/noteManager/decoratormods/placerToLauncherBackState.cpp

```diff
@@ -13,60 +14,60 @@
 // This file was automatically generated by the Team 302 code generator version 1.2.1.0
-// Generated on Monday, January 29, 2024 6:28:21 AM
+// Generated on Friday, February 2, 2024 7:00:51 PM
 
 // C++ Includes
 #include <string>
 
 // FRC includes
 
 // Team 302 includes
-#include "mechanisms/Thing1Mech/generated/Thing1Mech_rightFrontCW_StateGen.h"
-#include "mechanisms/Thing1Mech/decoratormods/Thing1Mech_rightFrontCW_State.h"
+#include "mechanisms/noteManager/decoratormods/placerToLauncherBackState.h"
 #include "teleopcontrol/TeleopControl.h"
 #include "teleopcontrol/TeleopControlFunctions.h"
 #include "utils/logging/Logger.h"
 
 // Third Party Includes
 
 using namespace std;
+using namespace noteManagerStates;
 
 /// @class ExampleForwardState
 /// @brief information about the control (open loop, closed loop position, closed loop velocity, etc.) for a mechanism state
-Thing1MechrightFrontCWState::Thing1MechrightFrontCWState(std::string stateName,
-														 int stateId,
-														 Thing1MechrightFrontCWStateGen *generatedState,
-														 Thing1Mech *mech) : State(stateName, stateId), m_genState(generatedState), m_mechanism(mech)
+placerToLauncherBackState::placerToLauncherBackState(std::string stateName,
+													 int stateId,
+													 noteManagerAllStatesStateGen *generatedState,
+													 noteManager *mech) : State(stateName, stateId), m_genState(generatedState), m_mechanism(mech)
 {
 }
 
-void Thing1MechrightFrontCWState::Init()
+void placerToLauncherBackState::Init()
 {
-	Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("Thing1MechrightFrontCWState"), string("init"));
+	Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("placerToLauncherBackState"), string("init"));
 
 	m_genState->Init();
 }
 
-void Thing1MechrightFrontCWState::Run()
+void placerToLauncherBackState::Run()
 {
-	// Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("Thing1MechrightFrontCWState"), string("run"));
+	// Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("ArrivedAt"), string("placerToLauncherBackState"), string("run"));
 	m_genState->Run();
 }
 
-void Thing1MechrightFrontCWState::Exit()
+void placerToLauncherBackState::Exit()
 {
 	m_genState->Exit();
 }
 
-bool Thing1MechrightFrontCWState::AtTarget()
+bool placerToLauncherBackState::AtTarget()
 {
 	auto attarget = m_genState->AtTarget();
 	return attarget;
 }
 
-bool Thing1MechrightFrontCWState::IsTransitionCondition(bool considerGamepadTransitions)
+bool placerToLauncherBackState::IsTransitionCondition(bool considerGamepadTransitions)
 {
 	// To get the current state use m_mechanism->GetCurrentState()
 
-	return (considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::AUTO_STAGE));
+	return (considerGamepadTransitions && TeleopControl::GetInstance()->IsButtonPressed(TeleopControlFunctions::EXAMPLE_MECH_FORWARD));
```

Comment:
Why is this using an ExampleMech teleop control?

Reply:
Base generated code as an example not mapped to anything

Reply:
issue 320 has been written to update this. Not linking so that it doesn't accidently get closed